AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1702,10 +1702,6 @@ - (void)__setSupernode:(ASDisplayNode *)newSupernode
       [self exitHierarchyState:stateToEnterOrExit];
     }
   }
-    
-  if ([newSupernode supportsUpwardPropagation]) {
```

Comment:
I think removing this makes sense, because it is not intended for sub nodes to pass layout options to their parents. Indeed, it is common to have multiple subnodes, so the value of supports upward propagation is misleading in this case.